lib/util.js

```diff
@@ -374,3 +374,61 @@ exports.parseAbbreviatedNumber = string => {
   }
   return null;
 };
+
+/**
+ * Match begin and end braces of input JSON, return only json
+ *
+ * @param {String} mixedJson
+ * @return {String}
+*/
+// eslint-disable-next-line consistent-return
+exports.cutAfterJSON = mixedJson => {
+  let open, close;
+  if (mixedJson[0] === '[') {
+    open = '[';
+    close = ']';
+  } else if (mixedJson[0] === '{') {
+    open = '{';
+    close = '}';
+  }
+
+  if (!open) {
+    throw new Error(`can't cut unsupported JSON (need to begin with [ or { ) but got ${mixedJson[0]}`);
```

Comment:
use `throw Error` and capitalize the message (there isnt' an eslint rule for this)

also put a colon next to "got" for more clarity